tests/__init__.py

```diff
@@ -1,7 +1,7 @@
 import os
 import unittest
 
-from tests.emulators import GoogleCloudStorageEmulator
+from octue.utils.cloud.emulators import GoogleCloudStorageEmulator
```

Comment:
Are the tests installed in production? The object that `GoogleCloudStorageEmulator` wraps is only included in the dev requirements

Reply:
Oh I see what you mean now. I'll move it into the `tests` package. Although I originally put it here because I thought people might want to import and use the emulator, which I was intending to do in `data-gateway`. As this is an SDK, what do you think?

Reply:
Oh, I see what you mean. I think if ppl want to do that they can add it as their own dependency. Maybe we add it later so that we can provide an easy test framework for folks, but for now I'd rather err on the side of keeping the library smaller.